portal-web/test/functional/com/liferay/portalweb/macros/APIAuthentication.macro

```diff
@@ -1,5 +1,43 @@
 definition {
 
+	macro assertConfigInSystemSettings {
```

Comment:
Since we are re-using this macro to perform assertions, it shouldnt be in this file. If this were to be written in Java, and we assumed the original testcase was FileInstallTest.java and the APIAuthConfiguration we were using would be a util like APIAuthTestUtil.java, we wouldnt add this as a function in APIAuthTestUtil.java because other parties that are using APIAuthTestUtil (or in this Poshi case, APiAuthentication) would almost never use it.

Instead, we would group this in our own testcase file, our in our case FileInstall.macro since this is the only file category that would reuse this function or macro.